CHANGES.rst

```diff
@@ -14,6 +14,9 @@ New features:
 
 Bug fixes:
 
+- Ensure error message is an unicode object
+  [mtrebron]
+
 - *add item here*
```

Comment:
@mtrebron usually this line is removed when the first change log entry on that category is added